modules/shared_vpc_access/main.tf

```diff
@@ -47,19 +50,19 @@ resource "google_compute_subnetwork_iam_member" "service_shared_vpc_subnet_users
   provider = google-beta
   count    = var.grant_services_network_role ? length(local.subnetwork_api) : 0
   subnetwork = element(
-    split("/", local.subnetwork_api[count.index][1]),
+    split("/", split(",", local.subnetwork_api[count.index])[1]),
```

Comment:
Not sure if this works but if we made the output into a list of maps, we could avoid the nested split - something like 

```tf
for i, api in local.active_apis : [for i, subnet in var.shared_vpc_subnets : {api: api, subnet: subnet}]
```

```tf
element(
    split("/", local.subnetwork_api[count.index].subnet),
    index(
      split("/", local.subnetwork_api[count.index].subnet),
      "subnetworks",
    ) + 1,
  )
  ```